templates/example_neighborhood_analysis_script.ipynb

```diff
@@ -18,7 +18,7 @@
   },
```

Comment:
<p>Just gold/Au</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/832/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>Au doesn't exist in the Example Dataset.</p>

Reply:
Nope, in that case we don't need to exclude anything